packages/client/src/components/pages/muster-page/muster-page.tsx

```diff
@@ -448,6 +419,34 @@ export const MusterPage = () => {
     setRosterToDateIso(range.toDateIso);
   };
 
+  const renderUnitComplianceTableRows = () => {
```

Comment:
I would prefer this React component to be stored in its own file and **props** used to pass down the data. It might be too much work do actually do this.

Reply:
I will address the component refactors in separate PR